hedera-node/hedera-file-service-impl/src/test/java/com/hedera/node/app/service/file/impl/test/handlers/FileCreateTest.java

```diff
@@ -287,6 +296,17 @@ void handleThrowsIfAttributeValidatorFails() {
         assertTrue(fileStore.get(FileID.newBuilder().fileNum(1234L).build()).isEmpty());
     }
 
+    @Test
+    void handleThrowsIfKeysSignatureFailed() {
```

Comment:
Might be good to have one of these for each of the handlers...

Reply:
done thank you